docs/dev/code_review.md

```diff
@@ -0,0 +1,121 @@
+- Does the nomenclature make sense? 
+- Are there any cases they may need to reconsider? 
+- Would comments or documentation make sense to the general user?
+
+## How to speed up code reviews
+
+Here are some tips and tricks:
+
+- Dedicated a certain amount of time and just review what you can in that time period 
+- Scan for missing doc strings and obvious mistakes 
+- Focus on what has been changed 
+- If an issue repeats then make a comment for them to fix the rest and add in the general feedback when submitting the comments 
+- Do not get too pedantic!
+- Break up the reviews if you can only review a few files or functions then do it and mention it. 
+
+## Reviewing a review 
+  
+As a developer you will need to address any comments that come up from code reviews like you would with reviewers comments in a paper submission. 
+Comments should addressed and resolved before merging the code. 
+
+If you do step 2 and look at the comments in `files changed` view you can address multiple comments in one batch. 
+
+### Batch Committing of Suggestions 
+
+First look for all the suggestion comments and if you agree then add to `batch suggestions`, once you have gone through all them 
+then at the top in the middle there is `commit suggestions` and this commit all the suggestions for you. 
+
+### Responding to Comments 
+
+If you don't agree with a suggestion try to find a compromise or acknowledge their effort and explain why you prefer your style.
+Remember to hit `Start a review` to prevent multiple email notifications. 
+
+If you made the change requested, then select `resolve conflict` so they know you addressed their comment.
+
+If you are answering a question or responding to some feedback, remember to acknowledge their time and effort, then address it in collaborative manner. 
+Remember to hit `Start a review` to prevent multiple email notifications. 
+
+Once all comments are addressed then follow steps 9-10 in the `How to start a Code Review` section. 
+
+## General Guidelines
+
+Here are some general guidelines to follow when code reviewing or responding to one: 
+
+- Dedicate time to review in a given period, avoid being distracted 
+- Ask questions instead of stating 
+- Remember to comment on the code 
+- Acknowledge time spent and say "Thank you" 
+- Suggest solutions or help
+- Avoid aggressive terminology 
+- Avoid reviewing in a bad mood or feeling pressured/rush 
+- Review all comments to make sure it sounds helpful and not demeaning 
+- Avoid pedantic points, focus on what is needed and not personal preferences 
+- Any amount of reviewing small or large is welcome! 
+- Approve! If things look good approve the PR! Approvals take us one step closer to being able to merge the code!
+
+!!! Note
+    If you approve the PR and you are the second to approve, or there are already 2 approvals and all concerns are addressed 
+    please merge the PR! Unless it is a release PR or a PR specifically looking for more than two approvals. 
```

Comment:
As mentioned previously, perhaps a clarification on the different types of reviews should be discussed.  Two approvals of the format/syntax of the code, with 0 approvals on the testing of the code is not ideal.

Reply:
Take a look at my changes and resolve the convo if it looks good.